lib/windowsbasetest.pm

```diff
@@ -50,7 +50,11 @@ sub open_powershell_as_admin {
         wait_screen_change { assert_and_click('window-max', timeout => 15) };
         assert_screen 'windows_server_powershel_opened', 30;
     } else {
-        send_key_until_needlematch 'user-acount-ctl-allow-make-changes', 'shift-a';
+        assert_and_click 'quick-features-menu';
+        assert_screen(['user-account-ctl-hidden', 'user-acount-ctl-allow-make-changes'], 120);
+        if (match_has_tag('user-account-ctl-hidden')) {
+            assert_and_click 'user-account-ctl-hidden';
+        }
```

Comment:
i might miss something but i did not find any match for `user-account-ctl-hidden`. i suspect that you try to do some trick here in order to perform the click. otherwise why not just 
`assert_and_click 'user-acount-ctl-allow-make-changes'`
i also assume that the `user-account-ctl-hidden` is a new needle. But not found in the PR for [osd](https://gitlab.suse.de/openqa/os-autoinst-needles-sles/-/merge_requests/1453) neither [o3](https://github.com/os-autoinst/os-autoinst-needles-opensuse/pull/695).

Reply:
A bit of trick to handle this [situation](http://kepler.suse.cz/tests/2313#step/prepare_wsl_feature/7), when the auth manager does not appear as the main window.  Happens occasionally only, I guess it is dependent on the workload of currently used worker.

Reply:
I would like to just ask you to make it one-liner if you do not mind.